components/indexer/component.go

```diff
@@ -238,6 +241,22 @@ func run() error {
 	return nil
 }
 
+func setupRoutes() {
+	deps.Echo.GET(api.RouteHealth, func(c echo.Context) error {
+		status, err := deps.Indexer.Status()
+		if err != nil {
+			return c.NoContent(http.StatusServiceUnavailable)
+		}
+
+		nodeStatus := deps.NodeBridge.LatestCommitment()
+		if status.CommittedSlot == nodeStatus.CommitmentID.Slot() {
+			return c.NoContent(http.StatusOK)
+		}
```

Comment:
There should be a little threshold, otherwise the indexer health switches from healthy to unhealthy for a short moment, if a new slot is committed.

See the health check in hornet maybe.
```go

const (
	isNodeAlmostSyncedThreshold = 2
)

...

isNodeAlmostSynced = confirmedIndex >= (latestIndex - isNodeAlmostSyncedThreshold)
```